client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -39,20 +38,28 @@ const resultsSpeak = debounceSpeak( { message: 'Search results loaded.' } );
 const errorSpeak = debounceSpeak( { message: 'No search results found.' } );
 
 function HelpSearchResults( {
-	currentUserId,
 	externalLinks = false,
-	hasAPIResults = false,
-	hasPurchases,
-	isSearching = false,
 	onSelect,
 	onAdminSectionSelect = noop,
 	searchQuery = '',
-	searchResults = [],
-	sectionName,
-	translate,
 	placeholderLines,
-	track,
 } ) {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const currentUserId = useSelector( ( state ) => getCurrentUserId( state ) );
+	const hasPurchases = useSelector( ( state ) =>
+		hasCancelableUserPurchases( state, getCurrentUserId( state ) )
+	);
+	const sectionName = useSelector( ( state ) => getSectionName( state ) );
```

Comment:
This can be written as `useSelector( getSectionName )` -- `useSelector` guarantees to call the selector with a single argument, `state`.

Reply:
addressed in https://github.com/Automattic/wp-calypso/pull/55753/commits/a5c0545a5f0624f70ca0e132d64a0f27b61a9fdb